src/styles/config.scss

```diff
@@ -295,17 +295,24 @@ $form-field-disabled-border: $border-width solid $border-color-light !default;
 // ----- checkbox (used with Checkbox component) -----
 $checkbox-dimension-small: 3 * $unit !default;
```

Comment:
this is kind of themeing related. it the varialbe is not set from other places, this default value will be used.